index.html

```diff
@@ -301,15 +289,39 @@ <h3><strong>About project name</strong></h3>
     <br />
     <!-- footer -->
     <footer>
-      <div class="mui-container mui--text-center mui--text-bottom">
-        <a href="https://github.com/DiLiubarets/project-1">
-          View project code on Github<br /><i
-            class="fa fa-github"
-            aria-hidden="true"
-          ></i
-        ></a>
+      <div id="about-project" class="mui-container mui--text-left">
+        <div class="mui--text-body2">
+          <h3><strong>About LSD</strong></h3>
+          <p>
+            LSD is an exchange-neutral tool designed for the modern cryptocurrency/token traders. 
+            Whether you are a professional investor or an amateur enthusiast, 
+            LSD provides all the necessary tools to grab the market bull by its horns.
+            <ul style="list-style-type:none;">
+              <li><i class="fa fa-bitcoin"></i> Track real-time prices to outmaneuver your opposition</li>
+              <li><i class="fa fa-bitcoin"></i> Predict the future with our database of historical prices</li>
+              <li><i class="fa fa-bitcoin"></i> Relax by subscribing to our notification system emailing you of changing trends</li>
+              <li><i class="fa fa-bitcoin"></i> View how your crypto stacks up against the competition by staying informed through our news section</li>
+            </ul>  
+            Powered by CoinCap and NY Times, LSD brings you accurate performance statistics for free!
+          </p>
+        </div>
+      <!-- </div> -->
+      <!-- <div class="mui-container mui-footer mui--text-center mui--text-bottom"> -->
+        <div class="mui-divider">
+        <a class="aGithub" href="https://github.com/DiLiubarets/project-1">
+          <i class="fa fa-github" aria-hidden="true"></i> View Project Code on Github
+        </a>
+        <!-- <a class="aCoin" href="https://docs.coincap.io/?version=latest">
+          <img src="coincap-logo-grey.jpg" style="width: 200px">
+          View API on CoinCap
+        </a> -->
+        <!-- <a class="aNY" href="https://developer.nytimes.com/">
+          View API on NYTimes
+        </a> -->
```

Comment:
API what I use for News its https://newsapi.org. its all on Slack. Liam need to few it. I believe he use the same API